mod_reforged/hooks/config/strings.nut

```diff
@@ -1097,6 +1098,19 @@ foreach (vanillaDesc in vanillaDescriptions)
 			]
 		}]
  	}),
+	RF_Decisive = ::UPD.getDescription({
+		Fluff = "There is no time to wait!",
+		Effects = [{
+			Type = ::UPD.EffectType.Passive,
+			Description = [
+				"Gain a stack of decisiveness whenever you end your turn without having used \'Wait\', up to a maximum of 3 stacks.",
+				::MSU.Text.colorGreen("+15") + " [Resolve|Concept.Bravery] while you have atleast 1 stack.",
+				"Skills build up " + ::MSU.Text.colorGreen("15%") + " less [Fatigue|Concept.Fatigue] while you have atleast 2 stacks.",
+				"Deal " + ::MSU.Text.colorGreen("15%") + " more Damage while you have 3 stacks.",
```

Comment:
`damage` (don't capitalize D)

Reply:
when do I capitalize terms and when don't I do that?

Damage is a very common battle brothers stat/term. If the word was fluff and used to describe something like an injury or a trigger condition ("when you deal any damage"), then I understand writing it lower case.